src/ape/utils/basemodel.py

```diff
@@ -35,6 +36,14 @@
     from ape.pytest.runners import PytestApeRunner
 
 
+class classproperty(object):
```

Comment:
There's nothing in Python stdlib that can acheive this?

Reply:
no, and also this wont work with setters. setters is where things seem to get complicated, which is why im guessing they dont have this

in our case , we only need a getter though